thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -154,7 +162,7 @@ body {
 .te-label {
   font-weight: normal;
   color:  app-shade(black, 9);
-  margin-bottom: 12px;
+  margin: 15px 0 12px 0;
```

Comment:
Blocking comment: Again this seems to be a global change to all `te-label`. Please create a specific class for your feature, unless this change is driven by design

Reply:
ah, yes - I changed this in the previous PR.